int256.go

```diff
@@ -337,3 +363,12 @@ func (z *Int) And(x, y *Int) *Int {
 	z = MustFromBig(big)
 	return z
 }
+
+// setZero sets default `0` value if `z` is nil
+func (z *Int) setZero() *Int {
```

Comment:
the naming is confusing a bit. Moreover, function `set` but returning int

Reply:
two names above is okay for me

Reply:
I have made the respective changes, please review.